client/client-multi/client-c/src/kaa/kaa_common_schema.h

```diff
@@ -38,7 +38,7 @@ extern "C" {
 
 
 typedef void (*serialize_fn)(avro_writer_t writer, void *data);
-typedef void *(*deserialize_fn)(void);
+typedef void *(*deserialize_fn)(); // NOTE: Do not add void in the parameter list.
```

Comment:
Added a comment.
I suggest to discuss improvements of Avro C and we will open a story. Ok?